graylog2-web-interface/src/components/common/ExpandableListItem.jsx

```diff
@@ -1,9 +1,83 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
 import { Checkbox } from 'components/graylog';
 import Icon from './Icon';
 
-import style from './ExpandableListItem.css';
+const ItemWrap = styled.li(({ padded }) => `
```

Comment:
Lets use `.css` here and in line 33 + 68 as well.

Reply:
After talking with @kyleknighted I understand the reason to not use styled components `css`. Possible performance improvements > syntax highlighting.